cmd/monitor/streaming/conf/clickhouse/spans/ddl_create_default_tables.sql.tpl

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE IF NOT EXISTS <database>.spans_series ON CLUSTER '{cluster}'
```

Comment:
missing conf section of `clickhouse.table.initializer@span`  in streaming bootstrap.yaml